packages/commitlint-issue-reference/.editorconfig

```diff
@@ -0,0 +1,16 @@
+# http://editorconfig.org
```

Comment:
Shouldn't this file move to the root of the directory. All (future) packages would benefit from the same code style.

Reply:
Yes :)